Makefile

```diff
@@ -54,48 +54,46 @@ helm.clean:
 
 e2e:
 	-helm package helm/portieris
-	@go test -v ./test/e2e --helmChart $$(pwd)/portieris-$(VERSION).tgz
```

Comment:
Is the `--helmChart` no longer needed?

Reply:
with helm 3, we don't think it is a requirement (we being me and @tim-gp)

Reply:
ok thanks